src/main/java/com/softserve/kickscooter/kickscootervehicle/management/controller/StatusController.java

```diff
@@ -0,0 +1,65 @@
+package com.softserve.kickscooter.kickscootervehicle.management.controller;
+
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterStatusDto;
+import com.softserve.kickscooter.kickscootervehicle.management.exceptions.ScooterIsDeadException;
+import com.softserve.kickscooter.kickscootervehicle.management.model.ScooterStatus;
+import com.softserve.kickscooter.kickscootervehicle.management.service.StatusService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@RequestMapping("/scooters/status")
+@AllArgsConstructor
+public class StatusController {
+
+    private StatusService statusService;
+
+    @ExceptionHandler({EntityNotFoundException.class, ScooterIsDeadException.class})
```

Comment:
exception handlers should be in controller advices

Reply:
Does ScooterIsDeadException mean not found?

Reply:
> Does ScooterIsDeadException mean not found?

Mean Scooter is present, but have status "BROKEN". I think we shouldn`t send to ui scooter with broken status